src/darwin/mod.rs

```diff
@@ -146,18 +112,34 @@ impl crate::Rasterize for Rasterizer {
         // Get loaded font.
         let font = self.fonts.get(&glyph.font_key).ok_or(Error::FontNotLoaded)?;
 
-        // First try the font itself as a direct hit.
-        self.maybe_get_glyph(glyph, font).unwrap_or_else(|| {
-            // Then try fallbacks.
-            for fallback in &font.fallbacks {
-                if let Some(result) = self.maybe_get_glyph(glyph, &fallback) {
-                    // Found a fallback.
-                    return result;
-                }
-            }
-            // No fallback, give up.
-            Err(Error::MissingGlyph(glyph.c))
-        })
+        // Find a font where the given char is presented.
+        let mut glyph_index = font.glyph_index(glyph.c);
+        let font = match glyph_index {
+            Some(_) => font,
+            None => font
+                .fallbacks
+                .iter()
+                .find(|font| {
+                    glyph_index = font.glyph_index(glyph.c);
+                    glyph_index.is_some()
+                })
+                .unwrap_or(font),
+        };
+
+        let is_missing_glyph = glyph_index.is_none();
+
+        // According to
+        // https://developer.apple.com/fonts/TrueType-Reference-Manual/RM07/appendixB.html
+        // the index of 0 must be a missing glyph character.
```

Comment:
```suggestion
        // The index of `0` always corresponds to the missing glyph character:
        // https://developer.apple.com/fonts/TrueType-Reference-Manual/RM07/appendixB.html
```

Just easier to say everything up front, then formatting doesn't get messed up by the URL.